### DISCUSSION TITLE
Releasing 7.43

### DISCUSSION THREAD

**Discussion by lordfolken** [XCSoar/XCSoar#1451] [General]
Hello, 

So I think we have one more bug to solve https://github.com/XCSoar/XCSoar/issues/1416. 
Unless there are objections, I would propose that [7.34](https://github.com/XCSoar/XCSoar/milestone/16) is being released. 

This poses a few challenges:
* The release process isn't documented 
* The former Maintainer went away with the signing keys, especially in regards to android. 

F-Droid: This is the easy case, as soon as the release is marked on github, F-droid will automatically build/sign/ add to repository the new Version 7.34. Users will receive an upgrade notification on F-Droid. Done.

Windows: afaik the releases where never signed, so not a problem

Kobo: Should be fine. 

Pi: standard debian package build

Android downloadedable from website: 
 - There will be a new signing key, so no upgrades to existing installations
 - Potentially we have to rename the .org.xcsoar, as this can only exist once with a given keypair. 

Android Playstore: 
A dev account is ready. However Playstore new apps have to supply a applicaiton bundle, and no longer an apk. This isnt' currently supported. 
The manifest name org.xcsoar is tied to the former maintainer's account/keys.  So this will also have to be changed. so no upgrades.

**Comment by kobedegeest** [comment]
#1349 would be nice to see included in next release. 

As to release for Android, the use of F-droid could be mentioned on the website maybe? If this methode is explained users could easily switch from playstore to F-droid for xcsoar. Along with a small note "For the moment XCSoar is not available in the playstore" or something. But even 7.42 is not available anymore so it is no reason to not release new version.
